bokehjs/src/coffee/models/tools/gestures/wheel_zoom_tool.coffee

```diff
@@ -51,13 +46,7 @@ class WheelZoomToolView extends GestureTool.View
 
     factor  = @model.speed * delta
 
-    zoom_info = scale_range({
-      frame: frame
-      factor: factor
-      center: [vx, vy]
-      v_axis_only: v_axis_only
-      h_axis_only: h_axis_only
-    })
+    zoom_info = scale_range(frame, factor, h_axis=h_axis, v_axis=v_axis, {x: vx, y: vy})
```

Comment:
This misunderstanding is a courtesy of dynamic typing and making assignment an expression in coffeescript.